src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -126,6 +126,36 @@ public HttpRequestData()
             Headers = new List<HttpHeaderData>();
         }
 
+        public static async Task<HttpRequestData> FromHttpRequestMessageAsync(System.Net.Http.HttpRequestMessage request)
+        {
+            var result = new HttpRequestData();
+            result.Method = request.Method.ToString();
+            result.Path = request.RequestUri.AbsolutePath;
```

Comment:
Is a null check `request.RequestUri != null` not needed?

Reply:
I see that `RequestUri` is `new Uri("http://example.com")` in all current calls of `FromHttpRequestMessageAsync` because of L30 in `HttpMessageHandlerLoopbackServer.cs`. Do we anticipate no further additions of `FromHttpRequestMessageAsync` calls with a null `RequestUri`

Reply:
I guess a null check can't hurt but you're right, in the current usages it should be always set.